.palantir/revapi.yml

```diff
@@ -1178,6 +1178,11 @@ acceptedBreaks:
       new: "class org.apache.iceberg.Metrics"
       justification: "Java serialization across versions is not guaranteed"
     org.apache.iceberg:iceberg-core:
+    - code: "java.class.removed"
+      old: "class org.apache.iceberg.MetadataUpdate.RemoveSnapshot"
+      justification: "Changing the RemoveSnapshot class to receive a list of snapshots\
+      \ IDs instead of a single snapshot ID. This will make the remove snapshots\
+      \ more efficient."
     - code: "java.method.removed"
```

Comment:
We shouldn't remove this, this is a still valid update in the protocol.

Reply:
I was wrong here, the protocol is already actually a [`RemoveSnapshots<list snapshots>`](https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L2833). This `RemoveSnapshot` is really more of how it's mapped internally ,and is unfortunately exposed to the public. I still think we'd need to keep this for library compatibility, but make the implementation less restrictive on only allowing 1 snapshot in the `RemoveSnapshots` list.